src/webextension/tablog.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
```

Comment:
hmm, I guess "chrome" is already defined as global while init WebExtension. Like startup is not defined in bootstrap.js